src/reducers/tasksReducer.js

```diff
@@ -163,6 +164,42 @@ export default function(state = initialState, action) {
       });
     }
 
+    case 'MOVE_TASK':
+      const { oldParentGid, newParentGid, taskGid } = action;
+      const newMap = Object.assign({}, state.taskMap);
+      const newParentTask = newMap[newParentGid];
+      const oldParentTask = newMap[oldParentGid];
+      const oldParentSubtaskGids = oldParentTask.subtask_gids.filter(
+        gid => gid !== taskGid
+      );
+      const newSubtaskGids = [...newParentTask.subtask_gids, taskGid];
+      const newSubtasks = newSubtaskGids.filter(
+        (gid, idx) => newSubtaskGids.indexOf(gid) === idx
+      );
+
+      newSubtasks.sort(function(t1, t2) {
+        const t1Date = new Date(newMap[t1].created);
+        const t2Date = new Date(newMap[t2].created);
+        if (t1Date === t2Date) {
```

Comment:
@colinfruit Is this the right comparison? We just want to return `0` immediately if they're the same, yes?

Reply:
I thought we were sorting by created then by id per the earlier comment. Did I misunderstand?

Reply:
@colinfruit Ooh I see, no you're right. Can you rename `t1` and `t2` to `gid1` and `gid2` please?

Reply:
I lost track of what was a task v. a date v. a gid, which is why I was confused.

Reply:
👍 will rename